docs/getting-started/network-page-overview.asciidoc

```diff
@@ -0,0 +1,207 @@
+[[network-page-overview]]
+= Network page overview
+
+The Network view provides key network activity metrics and an interactive map, facilitates investigation time enrichment, and provides network event tables that enable interaction with the Timeline. You can drag and drop items of interest from the Network view to Timeline for further investigation.
```

Comment:
`facilitates investigation time enrichment`

@jmikell821 
I know that's my fault (it's awful, and I've no idea what it means 😬), but please remove or change it - or remind me to when you're done. Thanks

Reply:
@benskelker I was kinda wondering what this meant - LOL! Will think of another sentence.